api-processor/build.gradle

```diff
@@ -11,5 +11,6 @@ dependencies {
     implementation 'org.springframework:spring-context'
 
     testImplementation project(':test-common')
```

Comment:
We can remove this since `test-common-blackduck` declares it as an `api` dependency.